docs/topics/gradle.md

```diff
@@ -1,41 +1,48 @@
 [//]: # (title: Gradle)
 
-<microformat>
-    <p>Minimum supported Gradle version: <strong>%minGradleVersion%</strong></p>
-    <p>Minimum supported Android Gradle plugin version: <strong>%minAndroidGradleVersion%</strong></p>
-</microformat>
+Gradle is a build system that helps to automate and manage your building process. It downloads specified dependencies,
+packages your code, and prepares it for compilation.
 
-In order to build a Kotlin project with Gradle, you should [apply the Kotlin Gradle plugin to your project](#plugin-and-versions)
-and [configure the dependencies](#configuring-dependencies).
+To build a Kotlin project with Gradle, you'll need to add a Gradle extension, [the Kotlin Gradle plugin](#apply-the-plugin),
```

Comment:
replace `extension` with `plugin`.

`extension` in terms of Gradle - is an optional part of the plugin